jupyter_server/extension/application.py

```diff
@@ -96,7 +98,9 @@ def _prepare_templates(self):
         self.initialize_templates()
         # Add templates to web app settings if extension has templates.
         if len(self.template_paths) > 0:
```

Comment:
could just be `if self.template_paths`

Reply:
I won't change it as it is not modified by this PR.